src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -52,7 +52,7 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
       df: DataFrame,
       indexConfig: IndexConfig,
       path: Path): IndexLogEntry = {
-    val absolutePath = PathUtils.makeAbsolute(path)
+    val absolutePath = PathUtils.makeAbsolute(path.toString, spark.sessionState.newHadoopConf())
```

Comment:
To minimize changes:
```suggestion
    val absolutePath = PathUtils.makeAbsolute(path, spark.sessionState.newHadoopConf())
```

Reply:
👍